components/director/internal/open_resource_discovery/validation.go

```diff
@@ -975,6 +983,9 @@ func validateCustomDescription(el gjson.Result) error {
 	if el.Get("customDescription").Exists() && el.Get("type").String() != custom {
 		return errors.New("if customDescription is provided, type should be set to 'custom'")
 	}
+	if el.Get("customDescription").Exists() && el.Get("type").String() == custom && len(el.Get("customDescription").String()) > MaxDescriptionLength {
```

Comment:
Check minimum length as well as the customDescription can exsist with 0 length. Something like:

```json
{
    "customDescription": ""
}
```